lib/reek/cli/input.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 require_relative '../source/source_locator'
+require_relative '../configuration/app_configuration'
```

Comment:
This line is not needed since you don't refer to the class defined there by name (it is always injected/defined by implementations of `#configuration`.